node/index.ts

```diff
@@ -8,14 +8,64 @@ import * as SignalClient from './libsignal_client';
 
 const SC = bindings('libsignal_client') as typeof SignalClient;
 
+export class PublicKey {
+  private readonly nativeHandle: SignalClient.PublicKey;
+
+  private constructor(handle: SignalClient.PublicKey) {
+    this.nativeHandle = handle;
+  }
+
+  static from_native_handle(handle: SignalClient.PublicKey): PublicKey {
```

Comment:
Nitpick: JavaScript camelCase, please. (Surprised this didn't show up in `yarn lint` or `yarn format`.)

Reply:
@EvanHahn-Signal yes please any help on the JS tooling/linting side is most welcome

Reply:
I'll make a note to follow up with y'all on this in 2021.